wiki/Contests/en.md

```diff
@@ -1,97 +1,153 @@
+<!-- Please check redirects after adding new sections with year numbers. -->
 [o!s]: /wiki/shared/mode/osu.png
 [o!t]: /wiki/shared/mode/taiko.png
 [o!c]: /wiki/shared/mode/catch.png
 [o!m]: /wiki/shared/mode/mania.png
+[GCrown]: /wiki/shared/GCrown.png "First Place"
+[SCrown]: /wiki/shared/SCrown.png "Second Place"
+[BCrown]: /wiki/shared/BCrown.png "Third Place"
 
 # Contests
 
-Various contests ran within the osu!community.
+This article lists contests that are not listed on the [website's contests page](https://osu.ppy.sh/community/contests) (mostly before late-2016 and other community-ran contests).
```

Comment:
There are a couple of contests (Aspire 2018, Best of 2016, etc.) listed on this page and web's.

Reply:
I'll remove them, but it looks like some of the entries in the Community Contests page are missing some information.